src/components/alphaPicker/alphaPicker.js

```diff
@@ -95,227 +106,228 @@ define(['focusManager', 'layoutManager', 'dom', 'css!./style.css', 'paper-icon-b
         element.focus = focus;
     }
 
-    function AlphaPicker(options) {
+    export class AlphaPicker {
+        constructor(options) {
 
-        var self = this;
-        this.options = options;
+            this.options = options;
 
-        var element = options.element;
-        var itemsContainer = options.itemsContainer;
-        var itemClass = options.itemClass;
+            const element = options.element;
+            const itemsContainer = options.itemsContainer;
+            const itemClass = options.itemClass;
 
-        var itemFocusValue;
-        var itemFocusTimeout;
+            let itemFocusValue;
+            let itemFocusTimeout;
 
-        function onItemFocusTimeout() {
-            itemFocusTimeout = null;
-            self.value(itemFocusValue);
-        }
+            function onItemFocusTimeout() {
+                itemFocusTimeout = null;
+                this.value(itemFocusValue);
+            }
 
-        var alphaFocusedElement;
-        var alphaFocusTimeout;
+            let alphaFocusedElement;
+            let alphaFocusTimeout;
 
-        function onAlphaFocusTimeout() {
+            function onAlphaFocusTimeout() {
 
-            alphaFocusTimeout = null;
+                alphaFocusTimeout = null;
 
-            if (document.activeElement === alphaFocusedElement) {
-                var value = alphaFocusedElement.getAttribute('data-value');
-                self.value(value, true);
+                if (document.activeElement === alphaFocusedElement) {
+                    const value = alphaFocusedElement.getAttribute('data-value');
+                    this.value(value, true);
```

Comment:
```
Uncaught TypeError: this.value is not a function
    at onAlphaFocusTimeout (alphaPicker.js:135)
```
`this` is `Window`. Occurred on TV layout only.

Reply:
Hm, could it be because it's a callback from a timeout?